mne/inverse_sparse/tests/test_mxne_optim.py

```diff
@@ -104,15 +104,15 @@ def test_l21_mxne():
 
 def test_tf_mxne():
     """Test convergence of TF-MxNE solver"""
-    alpha_space = 10.
-    alpha_time = 5.
+    alpha_space = 40.
+    alpha_time = 0.1
```

Comment:
Actually it's not only the name that has been changed. May be the doc for alpha and rho are misleading, should we change that @agramfort?
@Eric89GXL they are kind of different parameters: `alpha_space = alpha * (1 - rho)` and `alpha_time = alpha * rho`